.env.dist

```
@@ -0,0 +1,96 @@
+# https://symfony.com/doc/current/configuration.html#configuring-environment-variables-in-env-files
+
+PHPLIST_DATABASE_DRIVER=pdo_mysql
+PHPLIST_DATABASE_PATH=
+PHPLIST_DATABASE_HOST=127.0.0.1
+PHPLIST_DATABASE_PORT=3306
+PHPLIST_DATABASE_NAME=phplistdb
+PHPLIST_DATABASE_USER=phplist
+PHPLIST_DATABASE_PASSWORD=phplist
+DATABASE_PREFIX=phplist_
+LIST_TABLE_PREFIX=listattr_
+PHPLIST_ADMIN_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PHPLIST_ADMIN_PASSWORD key should go before the PHPLIST_DATABASE_DRIVER key

(UnorderedKey)